src/Aws/Api/Gateway.hs

```diff
@@ -253,3 +248,49 @@ responseBody = proxyResponseBody . at ()
 type Request = ProxyRequest ByteString
 
 type Response = ProxyResponse ByteString
+
+emptyProxyRequest :: ProxyRequest a
```

Comment:
Whats the context here?

Reply:
empty data constructors to facilitate lensed building:
```haskell
    request :: GW.ProxyRequest (Json.TextValue ByteString)
    request =
      emptyProxyRequest
        & GW.requestHttpMethod .~ "PUT"
        & GW.requestPath .~ "/products/versions/"
        & GW.requestHeaders .~ [("X-Api-Key", "ol0iBNHcGi7IJrTUlHj9q75v1gvyBDxE4O569jP3")]
        & GW.requestBody ?~ body
```